opensearch/src/main/java/org/opensearch/sql/opensearch/storage/script/filter/lucene/relevance/QueryQuery.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.sql.opensearch.storage.script.filter.lucene.relevance;
+
+import com.google.common.collect.ImmutableMap;
+import org.opensearch.index.query.QueryBuilders;
+import org.opensearch.index.query.QueryStringQueryBuilder;
+
+/**
+ * Class for Lucene query that builds the query_string query.
+ */
+public class QueryQuery extends NoFieldQuery {
+  /**
+   * Default constructor for QueryQuery configures how RelevanceQuery.build() handles
+   * named arguments by calling the constructor of QueryStringQuery.
+   */
+  public QueryQuery() {
```

Comment:
We attempted this, but had issues because `QueryStringQuery` is a `RelevanceQuery` and requires at least 2 arguments or else it throws a `SyntaxCheckException`, but the query function requires only 1 argument.

Reply:
Right, I tested this too.
But it is possible to move this to a `public static` field, which is accessible to both queries. And it works!
This could be stored in `QueryQuery` class, `QueryStringQuery` or even in `FunctionParameterRepository` - TBD

Reply:
Fixed in 9d1497503f73f2baaba63c7e637b712a92ed214c